pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/rbac/role_binding.go

```diff
@@ -43,6 +44,11 @@ func (f *RoleBinding) SetTemplateDefaults() error {
 const managerBindingTemplate = `apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
+  labels:
+    app.kubernetes.io/name: manager-rolebinding
+    app.kubernetes.io/component: manager
```

Comment:
```suggestion
    app.kubernetes.io/component: rbac
```

The same component is not the manager. It does not build the manager but the permissions / RBAC to allow the manager work.

Reply:
updated in `kustomize/v1` plugin in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/8a2c1d7c48ec7743828b14a04ae9de765cde30b2

Reply:
Should be finished in both `kustomize` plugins as of commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/daaca98d6775fe54fda2a0017d2f0f4b1369c1de

Marking this particular comment as resolved.